Source/WebKit/UIProcess/glib/DisplayLinkGLib.cpp

```diff
@@ -25,20 +25,34 @@
 
 #include "config.h"
 #include "DisplayLink.h"
+#include "Logging.h"
+#include <wtf/text/StringToIntegerConversion.h>
 
 namespace WebKit {
 
 using namespace WebCore;
 
 void DisplayLink::platformInitialize()
 {
+    if (const char* envString = getenv("WEBKIT_PROCESS_DISPLAY_LINK_THROTTLE")) {
```

Comment:
I would use a different name, I don't understand neither what this means nor what I'm supposed to pass just by looking at the name. It seems this receives a ratio that we use to calculate the refresh rate, right? Wouldn't it be easier for the user to pass the desired fps directly and then calculate the ratio? I guess it's because we don't really support any fps, but only a fraction of the actual one, right? In that case I would still use a better name also avoiding display link which is an internal name that it's not very clear in non apple ports. What about WEBKIT_DISPLAY_REFRESH_THROTTLE_RATIO?